valohai/internals/notebooks.py

```diff
@@ -0,0 +1,61 @@
+import json
+import os
+import shlex
+from typing import Union, List
+
+
+# TODO: This file is a copy-pasta from https://github.com/valohai/jupyhai
+# TODO: DRY between libs
```

Comment:
except for the `get_notebook_command`, but yes

the DRYing will happen after the merge/release